tsconfig.json

```diff
@@ -20,7 +20,8 @@
     "noUnusedParameters": true,
     "noFallthroughCasesInSwitch": true,
     "paths": {
-      "@/*": ["./src/*"]
+      "@/*": ["./src/*"],
+      "ipc/*": ["./src/ipc/*"]
```

Comment:
Why are we adding ipc to the routes? Personally, I prefer to only have `@` to keep things simple